common/logger.js

```diff
@@ -16,13 +16,14 @@ function transport() {
         });
     } else {
         return new transports.Console({
-            silent: !!process.env.TEST_SERVER === true
+            silent: !!process.env.TEST_SERVER === true,
+            format: format.simple()
         });
     }
 }
 
 module.exports = createLogger({
-    level: 'info',
+    level: isDev ? 'debug' : 'info',
     format: format.combine(format.errors({ stack: true }), format.json()),
```

Comment:
You could add `format.colorize()` to the `combine()` call here to get something similar to what `debug` did:
![image](https://user-images.githubusercontent.com/394376/60586700-7f66c500-9d8a-11e9-86ee-b113a93c274b.png)